bc_obps/service/data_access_service/facility_service.py

```diff
@@ -14,3 +15,7 @@ def get_all_facilities_for_user(cls, user: User) -> QuerySet[Facility]:
             # Industry users can only see operations associated with their own operator
             user_operator = UserOperatorService.get_current_user_approved_user_operator_or_raise(user)
             return queryset.filter(ownerships__operation__operator_id=user_operator.operator_id).distinct()
+
+    @classmethod
+    def get_currently_owned(cls, operation: Operation) -> QuerySet[Facility]:
```

Comment:
nit: Could we change the name of this function to be more relevant to its purpose? For example, `get_active_facilities_by_operation`.

Reply:
`active` might mean something different, I'll think of something